### ISSUE TITLE
do you have efi for sonoma? i have same mainboard.

### ISSUE THREAD

**Issue by mumin16** [chinh4thepro/Hackintosh-MSI-B650P-WIFI-Ryzen-7800X3D#1]
do you have efi for sonoma? i have same mainboard.

**Comment by chinh4** [comment]
Nope... I only hackintoshed for a bit to see what all the hype was really all about, but I just ended up going back to linux and posted my EFI here in case I want to test it out again...
